plugins/blend/README.md

```diff
@@ -0,0 +1,3 @@
+# blend
+
+TODO
```

Comment:
Greg I think you originally wrote this plugin. I'll add some quick documentation here.